modAionImpl/test/org/aion/zero/impl/types/A0BlockHeaderTest.java

```diff
@@ -129,6 +132,7 @@ public void testBlockHeaderFromUnsafeSource() throws Exception {
         assertThat(header.getSolution()).isEqualTo(new byte[1408]);
         assertThat(header.getNonce()).isEqualTo(ByteUtil.EMPTY_WORD);
         assertThat(header.getDifficulty()).isEqualTo(ByteUtil.EMPTY_HALFWORD);
+        assertThat(header.getVersion() == (byte) 0x01);
```

Comment:
style: assertThat(header.getVersion()).isEqualTo((byte) 0x01);

Reply:
the getVersion is returning byte, doesn't have the isEqualTo method.